source/release-notes/5.0.txt

```diff
@@ -895,6 +895,13 @@ Security Counter
     which reports the number of times a :dbcommand:`hello` request
     includes a valid :data:`hello.saslSupportedMechs` field.
 
+Repl
+  - :serverstatus:`repl` now includes a
+    :ref:`primaryOnlyServices<server-status-repl>` document which
```

Comment:
Two things:

- Should this go into the "Replication Metrics" section above this one, rather than creating a new "Repl" section?
- Should we link directly to the "primaryOnlyServices" document? As in:
```
:serverstatus:`~repl.primaryOnlyServices`
```

The tilde at the beginning trims everything before the field name, so it should render as "primaryOnlyServices" rather than "repl.primaryOnlyServices".

Reply:
This should be "document 'that' contains", not 'which'.

Reply:
I updated the link and the "which" -> "that", as for the repl output I feel it is more appropriate to leave it as is I feel like this would have come out in TR if it were not the case. I attached [a copy of serverStatus]([https://jira.mongodb.org/browse/DOCS-13871]) output to the ticket where metrics shows under a different document hierarchy.